### PR TITLE
BOP-323 Remove references to spec.components.core

### PR DESCRIPTION
**Pull request by ranyodh**

### CODE REVIEW

File: blueprints/kind-example/kind-example.yaml [hunk 1]
```diff
@@ -6,17 +6,6 @@ spec:
   kubernetes:
     provider: kind
   components:
-    core:
```
Comment (nwneisen) [inline blueprints/kind-example/kind-example.yaml:9]: These examples are just the output of `bctl init`. Can you check where they are being used and have the doc tell users to use the command instead? Then we can remove these and don't have to worry about them getting outdated.

Comment (ranyodh) [reply blueprints/kind-example/kind-example.yaml:9]: This particular example blueprint was not used anywhere. The docs calls to run `bctl init`. For `k0s` and `k0s-with-tf` example, the docs rely of these example bluepinrts.

So, I removed `kind-example.yaml`, but kept the other ones.

**Review by nwneisen — APPROVED** [review]
LGTM

**Review by sakshisharma84 — APPROVED** [review]
looks good!

**Pull request merged**